### PR TITLE
Testimonials & Milestones added

### PR DESCRIPTION
**Pull request by RockingSNP**
## Changes proposed
Update Profile

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers
Review this and help me to update this to my profile

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5415"></a>

### CODE REVIEW

**Review by TRohit20 — APPROVED** [review]

File: data/RockingSNP/testimonials/RockingSNP.json [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (MahirMahdi) [inline data/RockingSNP/testimonials/RockingSNP.json:1]: I think this addition made JSON check fail. Try removing it.

**Review by MahirMahdi — CHANGES_REQUESTED** [review]

**Comment by RockingSNP** [comment]
`Let it be merged`

**Comment by SaraJaoude** [comment]
> Let it be merged

@RockingSNP writing this is not necessary. All maintainers receive notifications - please do not forget that maintainers are in different timezones and have other commitments. 

I will merge your Pull Request but the point of Testimonials is not to give one to yourself, but to give one to other members of the community.  You may want to take a look at our Testimonials docs here: https://linkfree.eddiehub.io/docs/how-to-guides/testimonials

**Comment by RockingSNP** [comment]
> > Let it be merged
> 
> @RockingSNP writing this is not necessary. All maintainers receive notifications - please do not forget that maintainers are in different timezones and have other commitments. 
> 
> I will merge your Pull Request but the point of Testimonials is not to give one to yourself, but to give one to other members of the community.  You may want to take a look at our Testimonials docs here: https://linkfree.eddiehub.io/docs/how-to-guides/testimonials

Yes, next time I'll keep that in mind. & Thank You!

**Pull request merged**